conans/client/generators/cmake.py

```diff
@@ -32,6 +32,19 @@ def join_defines(values, prefix=""):
         self.src_paths = join_paths(cpp_info.src_paths)
 
         self.libs = join_flags(" ", cpp_info.libs)
+
+        framework_paths = join_paths(cpp_info.framework_paths)
+        self.find_frameworks = ""
+        for framework in cpp_info.frameworks:
+            var = "CONAN_FRAMEWORK_" + framework.upper()
+            find_framework = "find_library({var} {framework} " \
```

Comment:
I am not fully sure, may be there are guards like `if (APPLE)` needed to prevent frameworks from being searched on unsupported platforms?

Reply:
Yes, probably it is safer.

Reply:
now with sanity check